src/odemis/driver/test/pwrmccdaq_test.py

```diff
@@ -0,0 +1,318 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+'''
+Created on 21 Feb 2024
+
+@author: Stefan Sneep
+
+Copyright © 2024 Stefan Sneep, Delmic
+
+This file is part of Odemis.
+
+Odemis is free software: you can redistribute it and/or modify it under the terms
+of the GNU General Public License version 2 as published by the Free Software
+Foundation.
+
+Odemis is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
+PURPOSE. See the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along with
+Odemis. If not, see http://www.gnu.org/licenses/.
+'''
+import copy
+import logging
+import time
+import unittest
+
+from odemis import model
+from odemis.driver import pwrmccdaq
+
+logging.getLogger().setLevel(logging.DEBUG)
+
+CONFIG_MCC = {
+                "name": "Interlock test", "role": "bsd-interlock",
+                "mcc_device": "fake",
```

Comment:
Typically we put by default code that test the real hardware, and set the simulator mode when TEST_NOHW env variable == 1. Could you do the same here?